lib/src/bound_multipart_stream.dart

```diff
@@ -93,7 +93,7 @@ class BoundMultipartStream {
 
   Stream<MimeMultipart> get stream => _controller.stream;
 
-  late StreamSubscription _subscription;
+  late StreamSubscription<List<int>> _subscription;
```

Comment:
This is correct, but we should watch for downstream issues (rolls blocked now that the type info is more correct, and casts are no longer required, ...).

Reply:
given this is PRIVATE I'm not too worried.

We could make it `<void>` which is still valid, I think...